youths/schema.py

```diff
@@ -14,10 +14,11 @@
 from .consts import GENDERS, LANGUAGES
 from .enums import NotificationType
 from .models import YouthProfile
-from .utils import create_approval_token, mail_approval_link
 
 
 class YouthProfileType(DjangoObjectType):
+    gender = graphene.Field(graphene.String, source="gender")
```

Comment:
This field is not required in the django model so it can be blank. This raises an error because `graphene.Enum` expects it to have a value, if I understood correctly. This is a workaround I found from the discussion of [this issue](https://github.com/graphql-python/graphene-django/issues/474).

Reply:
what if you declare `YouthProfileType` after `Gender = graphene.Enum("Gender", [(g[0].upper(), g[0]) for g in GENDERS])`, and then use it for this `gender` field:
```
gender = Gender()
```
would that help with the non-required issue? I don't know for sure though 😅

Reply:
Luckily easy to test. Still gives the same error: `Expected a value of type \"Gender\" but received: `. Return data includes `"gender": null,`...

Reply:
With the workaround, the data returned is `"gender": "",`, but includes no errors.

Reply:
Even though this workaround gets rid of the error, it will probably cause a bit of trouble for the frontend: the underlying field is actually an enum with a couple of choices (+ no value), but the data returned is just a string. While the frontend will still provide the value as an enum to the mutation, I don't think it's clear how it should show the returned data.

Reply:
I suggest we leave it as is for now, record it as an issue and fix in a later PR.